mxcube3/routes/qutils.py

```diff
@@ -537,11 +542,12 @@ def set_dc_params(model, entry, task_data):
     acq.path_template.base_prefix = params['prefix']
 
     full_path = os.path.join(mxcube.session.get_base_image_directory(),
-                             params.get('path', ''))
+                             params.get('subdir', ''))
```

Comment:
`full_path` is always _full_.  I created `subdir` because it is the one the user can populate (by default is the sample_name) and it was not consistent in general. Before this change, there was the `path` which was meaning the full_path but also the sample_name. I noticed this when I reloaded the interface and it was joining the path with itself.
I renamed the property to subdir.